Makefile

```
@@ -26,9 +26,14 @@ ifneq ($(GOSUM),)
   ldflags += -X github.com/cosmos/cosmos-sdk/version.VendorDirHash=$(shell $(GOSUM) go.sum)
 endif
 
-ifeq ($(WITH_CLEVELDB),yes)
+ifeq ($(DB_BACKEND),cleveldb)
   build_tags += gcc
   ldflags += -X github.com/cosmos/cosmos-sdk/types.DBBackend=cleveldb
+else ifeq($(DB_BACKEND),leveldb)
+ # Do nothing as LevelDB is the default backendDB of Cosmos SDK
+else
+build_tags += gcc
+ldflags += -X github.com/cosmos/cosmos-sdk/types.DBBackend=rocksdb
```

I've tried the following three commands: 

1. `make install DB_BACKEND=cleveldb` 
2. `make install DB_BACKEND=leveldb`
3. `make install` 

Number 1 and 2 executed properly, but number 3 didn't. 

I think this is due to the spaces that are being put inside the branches conditions and this should fix it: 

```suggestion
ifeq ($(DB_BACKEND),cleveldb)
  build_tags += gcc
  ldflags += -X github.com/cosmos/cosmos-sdk/types.DBBackend=cleveldb
else ifeq ($(DB_BACKEND),leveldb)
  # Do nothing as LevelDB is the default backendDB of Cosmos SDK
else ifeq ($(DB_BACKEND),)
  build_tags += gcc
  ldflags += -X github.com/cosmos/cosmos-sdk/types.DBBackend=rocksdb
endif
```